app/auth/middleware.go

```diff
@@ -62,3 +63,20 @@ func UserIDMiddleware(sessionStore *database.SessionStore) func(next http.Handle
 		})
 	}
 }
+
+// SetAuthorizationHeaderFromQueryMiddleware is a middleware that copies an access token
```

Comment:
If you use appropriate development tools, you can set the header correctly, I don't like the idea of using completely different stuffs in dev, I have always put the Authorization headers myself as a developer. I don't think it push toward good dev practices, and we should as the other devs are not really used to.

Reply:
reverted this commit